src/main.py

```diff
@@ -24,8 +26,7 @@ def get_health_status():
 except ValueError as e:
     logger.error(e)
 
-
-@app.get("/exercises", status_code=200)
+@app.get("/exercises", status_code=200, response_class=PlainTextResponse)
```

Comment:
If you want this change deployed, the version has to be upped so that on merge to main a new container is uploaded to our storage.